### ISSUE TITLE
Gradle Build Taking too long after adding Images

### ISSUE THREAD

**Issue by arpitKumarWadhwa** [udacity/ud839_Miwok#48]
Gradle Build is taking too long after I added the images and updated the Java files and the WordAdapter. 
It's currently 21 minutes as I am writing this since I clicked on the run App button. I've lost all hope that the Gradle Build will finish this year.

Before these changes, Gradle Build usually finished in about 2-3 minutes.

What to do?

**Comment by Kvaibhav01** [comment]
Here's a solution:

1. Go to File -> Settings -> Build, Execution, Deployment -> Build Tools -> Gradle
2. Check the 'Offline work' under 'Global Gradle settings'

That will reduce build time by 90%.

![image](https://user-images.githubusercontent.com/11731837/31081327-28017a54-a7a9-11e7-9c1b-d4549f013cb7.png)

**Comment by arpitKumarWadhwa** [comment]
@Kvaibhav01 Thanks for your suggestion. I had done this the day I installed Android Studio. The problem was something else.
I tried running the app the next day and the Gradle Build finished in just 2 mins. Also, when I aborted the Gradle Build the previous day it gave me a error similar to "$ADB Key not set". That means for some reason the ADB/Debugger detached itself from my phone. That's why it wasn't able to finish the Gradle Build.
